AVR target is broken?

I've found that compiling for AVR target (arduino uno board) fails with following diagnostics:
`xpcc/src/xpcc/math/geometry/location_2d_impl.hpp:169:51: error: call of overloaded 'abs(float)' is ambiguous
    (std::abs(this->orientation - other.orientation) < __FLT_EPSILON__));`

Reproduction is very simple: clone getting-started-with-xpcc repo, change board in project.cfg to arduino_uno and then run scons.

--- a/src/xpcc/ui/display/graphic_display.cpp
+++ b/src/xpcc/ui/display/graphic_display.cpp
@@ -28,7 +28,7 @@
  */
 // ----------------------------------------------------------------------------
 
-#include <stdlib.h>
+#include <cstdlib>
 #include <xpcc/math/utils/bit_operation.hpp>
 
 #include "font/fixed_width_5x8.hpp"

